openid-4-verifiable-presentations-1_0.md

```diff
@@ -2810,6 +2825,91 @@ The following is a non-normative example of the unsecured payload of the Key Bin
 
 <{{examples/response/kb_jwt_unsecured.json}}
 
+### SD-JWT VCDM {#sd-jwt_vcdm}
+
+SD-JWT VCDM (Verifiable Credential Data Model) extends the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] credential format and allows to incorporate existing data models that use Linked Data, e.g., W3C VCDM [@?VC_DATA], while enabling a consistent and uncomplicated approach to selective disclosure.
+
+Information contained in SD-JWT VCDM credentials can be processed using a JSON-LD [@?JSON-LD] processor after the SD-JWT VC processing.
+
+When IETF SD-JWT VC is mentioned in this specification, SD-JWT VCDM defined in this section MAY be used.
+
+#### Format
+
+SD-JWT VCDM credentials are valid SD-JWT VC credentials and all requirements from [@!I-D.ietf-oauth-sd-jwt-vc] apply. Additionally, the requirements listed in this section apply.
+
+For backward compatibility with JWT processors, the following registered JWT claims MUST be used:
+
+* To represent the validity period of SD-JWT VCDM (i.e., cryptographic signature), `exp`/`nbf`/`iat` Claims encoded as a UNIX timestamp (NumericDate) MUST be used.
```

Comment:
I will respectfully request `iat` not be characterized as a claim having to do with the validity period of the [SD-]JWT[ VCDM].

Reply:
Will remove iat here.